uui/components/typography/Text.tsx

```diff
@@ -1,23 +1,42 @@
 import { withMods } from '@epam/uui-core';
 import { getTextClasses, TextSettings } from '../../helpers';
-import { Text as uuiText, TextProps as UuiTextProps } from '@epam/uui-components';
+import * as UuiComponents from '@epam/uui-components';
```

Comment:
```suggestion
import * as uuiComponents from '@epam/uui-components';
```
use small letters for namespaces